src/main/java/seedu/duke/commons/Messages.java

```diff
@@ -0,0 +1,13 @@
+package seedu.duke.commons;
+
+/**
+ * Container for user visible messages
+ */
+public class Messages {
```

Comment:
Good idea to use abstraction here but maybe you can consider using an enum instead :+1: